app/controllers/orders_controller.rb

```diff
@@ -15,20 +15,15 @@ def update
   end
 
   def index
-    status = ValidateStatusFilter.call(status: params[:status])
-    @show_status = status.show_status
-    result = OrdersQuery.call(user: current_user, status: status.valid_status)
+    result = ValidateStatusFilter.call(status: params[:status])
+    @show_status = result.status
+
+    result = OrdersQuery.call(user: current_user, status: result.valid_status)
```

Comment:
it would be better to rename these variables
for example 
```
result = ValidateStatusFilter.call(status: params[:status])
orders = OrdersQuery.call(user: current_user, status: result.valid_status)
```